handbook/engineering/organization.md

```diff
@@ -0,0 +1,35 @@
+# Organization
+
+The purpose of our engineering organization is to build a product that is valuable to our customers.
+
+To achieve this purpose, we need to ensure two things:
+
+1. Each feature or product that we build needs to coherently solve a particular customer problem. {#customer-needs}
+1. We make the necessary technology investments to maximize our velocity over time (e.g., building a standard component library, investing in automated testing). {#engineering-needs}
```

Comment:
```suggestion
1. We make the necessary technology investments to maximize our velocity over time (e.g. building a standard component library, investing in automated testing). {#engineering-needs}
```

Reply:
I used to never put commas after e.g. and didn't think anything of it until I saw some other teammates (including @sqs) [doing it](https://sourcegraph.com/search?q=r%3Asourcegraph%2Fsourcegraph+e.g.%2C+count%3A1000&patternType=literal). Someone went so far as to [open a PR](https://github.com/sourcegraph/careers/pull/62/files) on our job descriptions to change it to have a comma. Since then I have just started adding a comma.

Based on an internet search, it seems like different style guides have different recommendations on this, so there isn't a single "right" way. If we want to be consistent as a company then we should add it to our style guide: https://about.sourcegraph.com/handbook/communication/style_guide

I started a poll in Slack to inform that decision: https://sourcegraph.slack.com/archives/C02FSM7DU/p1587578988157700

Going to leave this as-is on this PR.